packages/react-core/src/demos/DataList/examples/DataListActionable.tsx

```diff
@@ -33,13 +33,13 @@ export const DataListActionable: React.FunctionComponent = () => {
 
   return (
     <DashboardWrapper mainContainerId="main-content-datalist-view-actions" breadcrumb={null}>
-      <PageSection>
+      <PageSection aria-label="Projects">
```

Comment:
Similar to what you did in the Banner demo file, we can instead use aria-labelledby linked to the Title component's id a couple lines down. Looks like other demo files may need a similar update as well